src/Language/Python/Validate/Scope.hs

```diff
@@ -1,10 +1,11 @@
 {-# language DataKinds, TypeOperators #-}
-{-# language GeneralizedNewtypeDeriving #-}
-{-# language TemplateHaskell, TypeFamilies, FlexibleInstances, MultiParamTypeClasses #-}
+{-# language DeriveFunctor #-}
 {-# language FlexibleContexts #-}
 {-# language RankNTypes #-}
 {-# language LambdaCase #-}
+{-# language OverloadedLists #-}
 {-# language ScopedTypeVariables, TypeApplications #-}
+{-# language MultiParamTypeClasses #-}
```

Comment:
How are you deciding whether to list extensions on multiple lines or not?

Reply:
They're loosely grouped by 'topic'. So scoped type variables and type applications are like 'extra system F annotations stuff'.